MaxText/layers/attentions.py

```diff
@@ -68,6 +68,12 @@
 # pytype: disable=attribute-error
 
 
+def validate_compute_axis_order(s: str) -> None:
```

Comment:
duplicate, remove. I see this method in pyconfig.py as well, which is right place to validate at the beginning of the run.

Reply:
Actually this is a different validate. Since the base.yml only allows string, so the pyconfig validates the flag's string value. In attention module, the axis_orders are really used as tuple, and it can be hard code / overwrite in different model class initiation, i.e. https://github.com/google/maxtext/blob/main/MaxText/layers/gpt3.py#L229

I don't want to risk with possible hard coded axis_orders passing from somewhere other than yaml flags.